src/typed-method-types/workflows/triggers/trigger-event-types.ts

```diff
@@ -0,0 +1,27 @@
+export const TriggerEventTypes = {
+  AppMentioned: "slack#/events/app_mentioned",
+  ChannelArchived: "slack#/events/channel_archived",
+  ChannelCreated: "slack#/events/channel_created",
+  ChannelDeleted: "slack#/events/channel_deleted",
+  ChannelRenamed: "slack#/events/channel_renamed",
+  ChannelShared: "slack#/events/channel_shared",
```

Comment:
In the original version of the SDK we had all of these fully typed, so that the shape of the payloads was defined by the type system (and people could get autocompletion on their triggers/filters). Are we planning to do that in the future? Or did we decide it's ultimately not worth it? 

Not a blocker in either case 👍

Reply:
We're planning to do that when we add support for `DefineTrigger` in the SDK!